Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,7 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $media->setProviderName($this->getRequest()->get(sprintf('%s', $this->getUniqid()), null, true)['providerName']);
```

Comment:
1. ```$this->getUniqid()``` is supposed to be set and as I have seen from the code, this was handled properly unless this is a hacking attempt and the admin intentionally leaves out the "token". So, the "token" sdlfkjsdf should always exist.
2. If we leave it as ```$this->getRequest()->get(sdlfkjsdf[providerName])``` then the result will always null since the get method ultimately will be called upon ParameterBag which uses in_array ``` return array_key_exists($key, $this->parameters) ? $this->parameters[$key] : $default;```
so in this case $this->parameters is an array that looks like
```
['sdlfkjsdf ' => 
                  ['providerName' => 'youtube',
                  'fileName-or-something-from-the-admin-form'=>'some-value' ]
'query_param1' => 'value1',
'query_param2' => 'value2'
 ]```
 so, there is no such key as sdlfkjsdf['providerName'] it's not how in_array works.

Not sure if a unit test for this is really necessary.